sql/2021/security/https_server_redirects.sql

```diff
@@ -0,0 +1,20 @@
+#standardSQL
+# Prevalence of server redirects from HTTP to HTTPS over Jan 2021 to Aug 2021
+SELECT
+  client,
+  date,
+  COUNT(DISTINCT url) AS total_http_urls_on_page,
```

Comment:
Is the name accurate?

```suggestion
  COUNT(DISTINCT url) AS total_http_urls_on_page_with_a_redirect,
```

Reply:
Good call, that name is indeed more accurate. However, I think the results would be more interesting if the actual `total_http_urls_on_page` would be used, so will rewrite the query.